app/abilities/ability.rb

```diff
@@ -6,6 +6,8 @@ class Ability
   def initialize(user)
     can :read, [:feature_query, :layers_pane, :legend_pane, :share_pane]
     can :read, [Node, Way, Relation, OldNode, OldWay, OldRelation]
+    can :read, RelationMember
+    can :read, OldRelationMember
```

Comment:
Are those actually "can read relation members if there's more than ten members"?

Reply:
By the way, one of the two new controllers was missing previously in the commit. Now added.